executable/SummarizeApp.re

```diff
@@ -46,6 +102,10 @@ let user =
         field("role", ~args=Arg.[], ~typ=non_null(role), ~resolve=(info, p) =>
           p.role
         ),
+        field(
+          "summary", ~args=Arg.[], ~typ=non_null(summary), ~resolve=(info, p) =>
+          p.summary
```

Comment:
here we'll do the aggregation and we'll return the `let summary = { unread: result, etc }`